.circleci/config.yml

```
@@ -84,6 +84,21 @@ jobs:
           name: Clippy
           command: cargo clippy --all-targets --workspace -- -D warnings
       - cache_save
+  doc:
+    docker:
+      - image: quay.io/influxdb/rust:ci
+    environment:
+        # Disable full debug symbol generation to speed up CI build
+        # "1" means line tables only, which is useful for panic tracebacks.
+        RUSTFLAGS: "-C debuginfo=1"
+    steps:
+      - checkout
+      - rust_components
+      - cache_restore
+      - run:
+          name: Cargo doc
+          command: cargo doc --workspace
```

I noticed that the `cargo doc` command took 2.5 minutes : https://circleci.com/gh/influxdata/influxdb_iox/14751?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

What would you think about building docs for only IOx and not also our (large and distinguished dependency tree)?

```suggestion
          command: cargo doc --no-deps --workspace
```

---

Done, slight improvement for `cargo doc...` from  2m59s to 1m54s. It doesn't get blazing fast because `doc` requires that all dependencies are `check`ed.